src/pages/Login/Login.js

```diff
@@ -1,10 +1,86 @@
-import React, { Component } from 'react';
+import React from 'react';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      pwd: '',
+    };
+  }
+
+  goToSinup = () => {
+    this.props.history.push('./Signup');
+  };
+
+  handlEmailInput = event => {
+    this.setState({
+      email: event.target.value,
+    });
+  };
+
+  handelPwdInput = event => {
+    this.setState({
+      pwd: event.target.value,
+    });
+  };
 
-export class Login extends Component {
   render() {
     return (
-      <div>
-        <h1>login</h1>
+      <div id="loginPage">
+        <div className="container">
+          <div className="Logo">
+            <img id="homeLogo" src="./images/home.png" alt="집꾸미기로고" />
+            <div className="logoName">집꾸미기 로그인</div>
+          </div>
+          <div className="viewbtnBox">
+            <button className="viewBtn1">
+              <a href="https://wecode.co.kr/">위코드</a>계정으로 로그인
+            </button>
+            <button className="viewBtn2">카카오톡 계정으로 로그인</button>
+            <button className="viewBtn3">네이버 계정으로 로그인</button>
+          </div>
+          <div className="textOr">
+            <span>or</span>
+          </div>
+          <div className="inputBox">
+            <input
+              className="email"
+              type="email"
+              placeholder="E-MAIL"
+              onChange={this.handlEmailInput}
+            />
+            <img id="emailIcon" src="./images/email.png" alt="이메일아이콘" />
+            <input
+              className="password"
+              type="password"
+              placeholder="PASSWORD"
+              onChange={this.handelPwdInput}
+            />
+            <img id="pwdIcon" src="./images/padlock.png" alt="비밀번호아이콘" />
+          </div>
+          <div className="btnBox">
+            <button
+              id="loginBtn"
+              type="submit"
+              disabled={
+                this.state.email.indexOf('@') !== -1 &&
+                this.state.pwd.length > 7
+                  ? false
+                  : true
```

Comment:
```suggestion
                !(this.state.pwd.length > 7)
```
이미 비교연산자의 평가값이 true or false 이므로 굳이 삼항연산자 없이 not 연산자 사용해서 표현가능합니다!